src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,22 @@
-export const TodoInfo = () => {};
+import React from 'react';
+import { UserInfo } from '../UserInfo';
+import { Todo } from '../../types/Todo';
+import { getUserById } from '../../services/getUserById';
```

Comment:
[CODE STYLE] - Don't use one-letter variable naming. It's better to use more descriptive names for variables, even in lambda expressions or function parameters. Consider renaming 'todo' to something more descriptive.